CONTRIBUTING.md

```diff
@@ -2,117 +2,63 @@
-      make sure to contact them to collaborate
-   3. If nobody has been assigned for the issue and you would like to work on it,
-      make a comment on the issue to inform the community of your intentions
-      to begin work
-   4. Follow standard Github best practices: fork the repo, branch from the
-      HEAD of `main`, make some commits, and submit a PR to `main`
+   1. If the issue is a proposal, ensure that the proposal has been accepted.
+   2. Ensure that nobody else has already begun working on this issue. If they have, make sure to contact them to collaborate.
+   3. If nobody has been assigned for the issue and you would like to work on it, 
+   make a comment on the issue to inform the community of your intentions
+   to begin work. Then we will be able to assign the issue to you. 
+   4. Follow standard GitHub best practices: fork the repo, branch from the
+   HEAD of `main`, make some commits, and submit a PR to `main`.
       - For core developers working within the ibc-go repo, to ensure a clear
         ownership of branches, branches must be named with the convention
-        `{moniker}/{issue#}-branch-name`
-   5. Be sure to submit the PR in `Draft` mode submit your PR early, even if
-      it's incomplete as this indicates to the community you're working on
-      something and allows them to provide comments early in the development process
-   6. When the code is complete it can be marked `Ready for Review`
-   7. Be sure to include a relevant change log entry in the `Unreleased` section
-      of `CHANGELOG.md` (see file for log format)
+        `{moniker}/{issue#}-branch-name`.
+   5. Feel free to submit the PR in `Draft` mode, even if
+   the work is not complete, as this indicates to the community you're working on
+   something and allows them to provide comments early in the development process.
+   6. When the code is complete it can be marked `Ready for Review`.
+   7. Be sure to include a relevant changelog entry in the `Unreleased` section
+   of [`CHANGELOG.md`](./CHANGELOG.md) (see file for log format).
 
 Note that for very small or blatantly obvious problems (such as typos) it is
 not required to an open issue to submit a PR, but be aware that for more complex
 problems/features, if a PR is opened before an adequate design discussion has
-taken place in a github issue, that PR runs a high likelihood of being rejected.
-
-Other notes:
-
-- Looking for a good place to start contributing? How about checking out some
-  [good first issues](https://github.com/cosmos/ibc-go/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-- Please make sure to run `make format` before every commit - the easiest way
-  to do this is have your editor run it for you upon saving a file. Additionally
-  please ensure that your code is lint compliant by running `golangci-lint run`.
+taken place in a GitHub issue, that PR runs a high likelihood of being rejected.
```

Comment:
I think we should just state that every pr should have an accompanied issue. I'm sure folks opening up typo pr's won't read this document